docs/conf.py

```diff
@@ -136,6 +136,15 @@
 # TODO(ddn): Get this from katana.libgalois.version
 copyright = "Katana Graph, Inc. 2022"
 
+rst_epilog = """
+.. |supports_compiled_operator| replace::
+    This method may be used in compiled operators with some restrictions.
+
+.. |lazy_compute| replace::
+    The information required to perform this operation efficiently is computed lazily on the first call to this
+    method. This information is shared between related methods when possible.
+"""
```

Comment:
I do not love this. I would much prefer to have directory level sets of replacements or something. Since there replacements are not global, they only apply to a few specific classes.

Reply:
I assumed the issue was that the doc strings are being used outside the scope of the original rst file:

`katana.local_native.PyCapsule.in_degree` isn't [katana.local.graph.rst](https://github.com/KatanaGraph/katana/pull/972/files#diff-74e0246d9861f9edeea27115f8a5a18470ffc8d61a50016fd600db92be689af1). I was assuming this gets pulled in because of subclassing or something.

If I am right, replacements are not directory scoped. An alternative would be to organize the replacement names themselves in some hierarchy.

Reply:
Yea. I bet you are right. And it probably is subclassing (distributed.Graph is a subclass of local.Graph). I'm not sure the best way to do this.

Reply:
I'm going to go ahead with this current hack; if y'all figure out something better, we can do it down the line.